python/eos/observable.py

```diff
@@ -78,3 +78,22 @@ def _repr_html_(self):
         result += r'</table>'
 
         return(result)
+
+    @staticmethod
+    def assert_valid_name(name):
+        obs_names = [ob[0] for ob in Observables()]
+        if name in obs_names:
+            pass
+        else:
+            raise ValueError("Observable with name '" + name + "' is not known")
+
+    @staticmethod
+    def assert_valid_name_and_kinematic(name, kinematic):
+        Observables.assert_valid_name(name)
```

Comment:
I would expect an assertion method to pass quietly if it is positive and raise an exception otherwise, but not to return a value.

Since a kinematic is tied to an observable and the operation is cheap and not used often, I simply assert again. It's the easiest code possible and perceptible performance does not take a dent.